experimental/css-has-pseudo/README-BROWSER.md

```diff
@@ -40,8 +40,72 @@ cssHasPseudo(document);
 <script>cssHasPseudo(document)</script>
 ```
 
+⚠️ Please use a versioned url, like this : `https://unpkg.com/@csstools/css-has-pseudo-experimental@0.1.0/dist/browser-global.js`
```

Comment:
```suggestion
⚠️ Please use a versioned url, like this : `https://unpkg.com/@csstools/css-has-pseudo-experimental@0.1.2/dist/browser-global.js`
```

Let's use latest

Reply:
This should also be generated :)
Made a note for this.